### PR TITLE
Fix story routes and Bugs on the Profile Page on Dark Mode , closes #167 , closes #166

### LINKED ISSUE #166: Dark mode bugs in users#show
Examples:

Bildschirmfoto 2024-03-30 um 19 31 44

### LINKED ISSUE #167: Story routes should be restful
The following routes don't feel intuitive:

```elixir
      live "/profile/create-story", StoryLive, :new
      live "/profile/edit-story/:id", StoryLive, :edit
```

It should be something like:

```elixir
      live "/profile/stories/new", StoryLive, :new
      live "/profile/stories/:id/edit", StoryLive, :edit
```

### PR DESCRIPTION
**Pull request by MICHAELMUNAVU83**
- I changed the routes for the stories to `/profile/stories/new` and `/profile/stories/:id/edit` .
- I also worked on the UI bug on the dark mode of the profile page , it is as such right now
Screenshot 2024-03-30 at 22 15 16

### CODE REVIEW

File: lib/animina_web/live/profile_live.ex [hunk 1]
```diff
@@ -120,25 +120,29 @@ defmodule AniminaWeb.ProfileLive do
...
+              </p>
+               <%= if @user.occupation && @user.occupation != "" do %>
+
+              <p class=" dark:text-gray-100  text-gray-600">
+                <%= gettext("I'm a ") %> <%= @user.occupation %>
+                
               </p>
+              <%end %>
 
-              <%= if @user.occupation && @user.occupation != "" do %>
```
Comment (wintermeyer) [inline lib/animina_web/live/profile_live.ex:131]: No need to replace this code. I don't want to display the occupation part if it is nil or empty.

File: lib/animina_web/live/profile_live.ex [hunk 2]
```diff
@@ -120,25 +120,29 @@ defmodule AniminaWeb.ProfileLive do
       <div :if={@user != nil}>
         <div class="flex items-center px-4 space-x-4 border border-gray-100 rounded-lg shadow-sm">
           <div class="py-4">
-            <h3 class="text-lg font-semibold"><%= @user.name %></h3>
-            <p class="text-sm font-medium text-gray-500">@<%= @user.username %></p>
```
Comment (wintermeyer) [inline lib/animina_web/live/profile_live.ex:124]: Please keep the @

File: lib/animina_web/live/profile_live.ex [hunk 3]
```diff
@@ -120,25 +120,29 @@ defmodule AniminaWeb.ProfileLive do
...
+
+            <h3 class="text-lg dark:text-white font-semibold"><%= @user.name %></h3>
+            <p class="text-sm dark:text-gray-100 font-medium text-gray-500"><%= @user.username %></p>
 
             <div class="mt-2">
-              <p class="text-gray-600">
-                <%= gettext("Lives in ") %> <%= @user.city.zip_code %> <%= @user.city.name %>.
```
Comment (wintermeyer) [inline lib/animina_web/live/profile_live.ex:128]: Please keep the `zip_code`.

**Review by wintermeyer — CHANGES_REQUESTED** [review]
It is ok for this time but I prefer to have PRs which tackle one problem and not multiple problems in the same PR. Small PRs are much easier and faster to review. Thx

**Comment by MICHAELMUNAVU83** [comment]
Okay @wintermeyer  , thanks

File: lib/animina_web/live/profile_live.ex (hunk 1 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/live/profile_live.ex:131]: I have made this change

File: lib/animina_web/live/profile_live.ex (hunk 2 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/live/profile_live.ex:124]: It is now there

File: lib/animina_web/live/profile_live.ex (hunk 3 above)
Comment (MICHAELMUNAVU83) [reply lib/animina_web/live/profile_live.ex:128]: The zip code is now there

**Pull request merged**